api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -599,9 +599,9 @@ public static String getOpenmrsProperty(String property) {
 	/**
 	 * Indicates the version of the search index. The index will be rebuilt, if the version changes.
 	 * 
-	 * @since 1.11
```

Comment:
Why did you change this?

Reply:
when we build the openmrs core, it will generate a person's Lucene index